staffeln/conductor/backup.py

```diff
@@ -261,6 +270,10 @@ def check_instance_volumes(self):
                             backup_id="NULL",
                             instance_id=server.id,
                             backup_status=constants.BACKUP_PLANNED,
+                            # Only keep the last 100 chars of instance_name and
+                            # volume_name for forming backup_name
+                            instance_name=server.name[-100:],
+                            volume_name=volume["name"][-100:],
```

Comment:
Is there any advantage to keeping the last 100 characters instead of the first one?
I prefer first 100 chars.